data-science-onramp/data-cleaning/clean_test.py

```diff
@@ -88,17 +128,16 @@ def get_blob_from_path(path):
 
 def test_clean():
     """Tests clean.py by submitting it to a Dataproc cluster"""
-
     # Submit job to Dataproc cluster
     job_client = dataproc.JobControllerClient(
         client_options={"api_endpoint": f"{CLUSTER_REGION}-dataproc.googleapis.com:443"}
     )
-    response = job_client.submit_job_as_operation(
-        project_id=PROJECT, region=CLUSTER_REGION, job=JOB_DETAILS
+    operation = job_client.submit_job_as_operation(
+        project_id=PROJECT_ID, region=CLUSTER_REGION, job=DATAPROC_JOB
```

Comment:
You don't need to supply these as keyword args: https://googleapis.dev/python/dataproc/latest/gapic/v1/api.html?highlight=submit%20job%20operation#google.cloud.dataproc_v1.JobControllerClient.submit_job_as_operation